docs/07-cli.md

```diff
@@ -1,31 +0,0 @@
-# Serverpod CLI
```

Comment:
Question: This seems to have been removed. Is the thought that the CLI should have all documentation through its help method?

I see a use-case for some additional information on the CLI commands that might need to be explained in more detail.